lib/etc/openturns.conf.in

```diff
@@ -614,18 +614,20 @@
   <LinearModelStepwiseAlgorithm-MaximumIterationNumber   value_int="1000"        />
 
   <!-- OT::GeneralLinearModelAlgorithm parameters -->
-  <GeneralLinearModelAlgorithm-KeepCovariance                 value_bool="true"      />
-  <GeneralLinearModelAlgorithm-OptimizeParameters             value_bool="true"      />
-  <GeneralLinearModelAlgorithm-UnbiasedVariance               value_bool="true"      />
-  <GeneralLinearModelAlgorithm-UseAnalyticalAmplitudeEstimate value_bool="true"      />
-  <GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound  value_float="1.0e-2"  />
-  <GeneralLinearModelAlgorithm-DefaultOptimizationScaleFactor value_float="2.0"     />
-  <GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound  value_float="1.0e2"   />
-  <GeneralLinearModelAlgorithm-MaximalScaling                 value_float="1.0e5"   />
-  <GeneralLinearModelAlgorithm-MeanEpsilon                    value_float="1.0e-12" />
-  <GeneralLinearModelAlgorithm-StartingScaling                value_float="1.0e-13" />
-  <GeneralLinearModelAlgorithm-DefaultOptimizationAlgorithm   value_str="TNC"     />
-  <GeneralLinearModelAlgorithm-LinearAlgebra                  value_str="LAPACK"  />
+  <GeneralLinearModelAlgorithm-KeepCovariance                   value_bool="true"      />
+  <GeneralLinearModelAlgorithm-OptimizeParameters               value_bool="true"      />
+  <GeneralLinearModelAlgorithm-UnbiasedVariance                 value_bool="true"      />
+  <GeneralLinearModelAlgorithm-UseAnalyticalAmplitudeEstimate   value_bool="true"      />
+  <GeneralLinearModelAlgorithm-DefaultUseMultiStart             value_bool="true"     />
+  <GeneralLinearModelAlgorithm-DefaultOptimizationLowerBound    value_float="1.0e-2"  />
+  <GeneralLinearModelAlgorithm-DefaultOptimizationScaleFactor   value_float="2.0"     />
+  <GeneralLinearModelAlgorithm-DefaultOptimizationUpperBound    value_float="1.0e2"   />
+  <GeneralLinearModelAlgorithm-MaximalScaling                   value_float="1.0e5"   />
+  <GeneralLinearModelAlgorithm-MeanEpsilon                      value_float="1.0e-12" />
+  <GeneralLinearModelAlgorithm-StartingScaling                  value_float="1.0e-13" />
+  <GeneralLinearModelAlgorithm-DefaultOptimizationAlgorithm     value_str="TNC"     />
```

Comment:
No. Don't use it. I explained why why so many times I can't remember (but at least twice during our last meeting). Use e.g. Cobyla instead.

Reply:
I did initially change it to Cobyla, but then Kriging examples stopped working, so I changed it back. I will try again and see what happens.

Reply:
Unit tests also have trouble with Cobyla:
```
387/547 Test  #674: pyinstallcheck_EfficientGlobalOptimization_std .........................***Failed    2.58 sec
Traceback (most recent call last):
  File "Z:\home\devel\project\python\test\t_EfficientGlobalOptimization_std.py", line 143, in <module>
    algo.run()
  File "Z:\home\devel\project\build\install\Lib\site-packages\openturns\uncertainty.py", line 547, in run
    return _uncertainty.EfficientGlobalOptimization_run(self)
TypeError: InvalidArgumentException : In CovarianceModelImplementation::setFullParameter, the component 0 of scale is non positive
```

```
366/547 Test  #998: pyinstallcheck_GeneralLinearModelAlgorithm_std .........................***Failed    5.03 sec
WRN - Warning! For coherency we set scale upper bounds = [15.6]
WRN - (previous message repeated 1 time)
WRN - Warning! For coherency we set scale upper bounds = [15.6,100]
WRN - Warning! The Cobyla algorithm failed. The error message is Maximum number of function evaluations reached
--- /home/devel/project/python/test/t_GeneralLinearModelAlgorithm_std.expout	2021-04-03 06:00:44.653731375 +0000
+++ /home/devel/project/build/python/test/t_GeneralLinearModelAlgorithm_std.out	2021-04-03 06:41:04.270457432 +0000
@@ -26,7 +26,7 @@
 ===================================================
 
 
-covariance (full optim)= AbsoluteExponential(scale=[0.1328], amplitude=[0.1907])
-trend (full optim)= [[-0.1034,1.014]]
+covariance (full optim)= AbsoluteExponential(scale=[22.1], amplitude=[1.756])
+trend (full optim)= [[0.1223,0.9941]]
 ===================================================
```